docs/getting-started/invite-members/README.md

```diff
@@ -34,7 +35,7 @@ In order to see a project and work in it, you have to be a member of a project.
 
 To view the list of **all project members and their roles** in the project, select Members in the project menu.
 
-![project-member-list](image-20210305192605943.png)
+![invite-members](Bildschirmfoto 2022-03-02 um 14.15.12.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.